packages/back-end/src/services/experiments.ts

```diff
@@ -718,8 +718,13 @@ export async function processPastExperiments(
         startDate: e.start_date,
         numVariations: 1,
         variationKeys: [e.variation_id],
+        variationNames:
+          e.variation_name == e.variation_id || !e.variation_name
+            ? []
+            : [e.variation_name],
```

Comment:
I think the code will be much simpler if we always fill this array.  Doesn't matter if the variation_name is falsy or the same as the id.

```ts
variationNames: [e.variation_name || ""],
```

Reply:
Yeah I was seeing an error in the code where because of "0" being a falsy value (which I didn't expect because technically the 0 is in quotes), there was a different value being displayed in the import experiment form for when the variation name equals the variation id (so it's also "0" and "1"). So, I was doing a lot of short-circuiting, eventually leading to this code. But I think that should get resolved by the new approach of using regex to check for numbers anyways... Also, I was wondering if it's okay if I make variation_name/experiment_name a definitive field, and not an optional field? Since, in reality, it is. There's always a variation_name/experiment_name field, it's just that sometimes the field is just another name for the ids. So, would it be okay to make them definitive fields or could that cause problems?